cloudformation/alarms.yml

```diff
@@ -0,0 +1,37 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: >
+  CloudFormation template for configuring CloudWatch alarms in the EPS account.
+  
+Parameters:
+  ConcurrencyThreshold:
+    Type: Number
+    Description: Threshold for the Lambda concurrency.
+    Default: 300
+    
+  EnableAlerts:
+    Type: String
+    AllowedValues: [ 'true', 'false' ]
+    Default: 'true'
+    Description: Whether to enable or disable alarms and notifications.
+  
+Resources:
+  EPSAccountConcurrencyAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties: 
+      AlarmName: EPS_Account_Concurrency_Alarm
```

Comment:
Is the intention here to use EPS as the value for stack in the slack message?

Also alarm names should follow the `<stackName>_<resource>_<metric>` to be properly parsed / be consistent, and don't need alarm in the actual name. So you might want to tweak it to be something like `Account_Lambda_Concurrency`

Reply:
I'll just pinch that name, it sounds intuitive to me.